subjects/road_intersection/README.md

```diff
@@ -65,15 +65,17 @@ You are free to decide what algorithm you want to implement for the traffic ligh
 
 Vehicles must obey this rules:
 
-- Vehicles must have a color depending on their route the colors are up to you to decide(ex:`r`- purple, `s`- Blue and `l`- Yellow). This must then be given during the audit
+- Vehicles must have a color depending on their route the colors are up to you to decide 
 
-- Autonomous, vehicles driving on a lane with a **given route** must follow the direction of
-  that route, its not possible for the driver to change lanes or route.
+- Vehicles must have a color so that it's possible to identify which route it will follow (ex:`r`- purple, `s`- Blue and `l`- Yellow). This information about the colors must be given to the auditor of the raid.
```

Comment:
Probably can merge line 68 and line 70.

https://github.com/01-edu/public/blob/332e922aaec357b2a8b5e9ecc25c8f929de95701/subjects/road_intersection/README.md?plain=1#L68
https://github.com/01-edu/public/blob/332e922aaec357b2a8b5e9ecc25c8f929de95701/subjects/road_intersection/README.md?plain=1#L70

Reply:
done